eclipse-theia/theia
# [vscode][proposed] Stub onWillCreateEditSessionIdentity

#### What it does

This event was recently added to proposed API `EditSessionIdentityProvider`. For now we add a stub that will allow `vscode.git@1.77.0` and more recent to activate and be successfully used in Theia applications.

Fixes #12521

#### How to test

You can use the vscode.git 1.77.0  extensions below: 
[git-.zip](https://github.com/eclipse-theia/theia/files/11481204/git-.zip)

- Extract the git extensions and copy them in the "plugins" folder. 
- start the example application, open a workspace
- `vscode.git` should activate, not complaining about `onWillCreateEditSessionIdentity` not existing. Built-in `git` will not ultimately work, unless `@theia/git` is removed from the app - but that's no necessary for this PR)

#### Review checklist


#### Reminder for reviewers

- As a reviewer, I agree to behave in accordance with [the review guidelines](https://github.com/theia-ide/theia/blob/master/doc/pull-requests.md#reviewing)
